pages/Persons.xml

```diff
@@ -225,10 +226,16 @@ guidelines for
 
          <div type="level1">
             <head>Commemoration</head>
-            <p>For Saints, you might wish to record the commemoration dates. Please
-               follow the following example. Please always include the source for the date (Sǝnkǝssār, Dǝggwā, EAe...).</p>
+            <p>For Saints, commemoration dates can be recorded as in the following examples. 
+               Please always include the source for the date (Sǝnkǝssār, Dǝggʷā, EAe...).</p>
+            <p>In some records, <att>type</att> with <val>diesNatalis</val> is used. This term is 
+            contested. Its use is not mandatory.</p>
```

Comment:
I would say not "contested" but specify that it is to be used only when the **commemoration is on the day of martyrdom**. One saint may have various dates on which he is remembered but there is only one `diesNatalis`.

I would remove "Its use is not mandatory" - rather, it is desirable when we are encoding the day of martyrdom.